candig_rnaget/api/operations.py

```diff
@@ -952,14 +949,16 @@ def search_continuous(format=None):
 
 def search_continuous_filters():
     """
-    TODO: Implement
+    :return: filters for continuous searches
     """
+    filter_file = pkg_resources.resource_filename("candig_rnaget", "orm/filter_continuous.json")
```

Comment:
can we rename the file to `filters_continuous.json`, to be consistent with the other two?